src/conf/data/_preprocessors_sliding_window.yaml

```diff
@@ -0,0 +1,14 @@
+# @package _global_
+_preprocessors:
+  spectrogram:
+    _target_: torchaudio.transforms.Spectrogram
+    n_fft: 1024
+    hop_length: 256
+  sliding_window_norm:
+    _target_: data.SlidingWindowNormalize
+    n_fft: 1024
```

Comment:
if it needs to be aligned with the spectrogram you can use ${_preprocessors.spectrogram.n_fft}